provider/openstack/legacy_networking.go

```diff
@@ -73,3 +75,13 @@ func (n *LegacyNovaNetworking) ResolveNetwork(name string) (string, error) {
 	}
 	return processResolveNetworkIds(name, networkIds)
 }
+
+// Subnets is part of the Networking interface.
+func (n *LegacyNovaNetworking) Subnets(instId instance.Id, subnetIds []network.Id) ([]network.SubnetInfo, error) {
+	return nil, errors.NotSupportedf("nova subnet")
```

Comment:
If we are going to do this here, then the discover spaces worker will need to be tweaked to handle this error and ignore it, since the worker thinks these methods should work. Add a todo also to the worker to revert the change when nova subnet support is added.
Should also add a test for this case.